thirdeye-detectionpipeline/src/main/java/ai/startree/thirdeye/detectionpipeline/operator/DelayOperator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.detectionpipeline.operator;
+
+import static ai.startree.thirdeye.spi.util.SpiUtils.optional;
+
+import java.time.Duration;
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class DelayOperator extends DetectionPipelineOperator{
+
+  private static final String DELAY_TIME = "delayTime";
+  private static final String RANDOM_OFFSET = "randomOffset";
+  @Override
+  public void execute() throws Exception {
+    Duration delayTime = Duration.parse(getPlanNode().getParams().get(DELAY_TIME).toString());
```

Comment:
Let's check with @cyrilou242 if there are any standard utils for doing this. There is `TimeUtils` I think which can help here.

Reply:
There are some utils for ISO-8601 Periods (P1D, P7D, etc...). 
If we plan sleeps of 1 day or more then we should use Periods to avoid timezones issues, but I understand this is not required.

In this case I'm fine with durations. I don't think there are utils for this. Though I think jackson is able to parse millis as duration automatically. So if a spec is introduced with typed params then the parsing should be done automatically. 
(not sure though, to test, and this makes the code heavier)

Reply:
Yeah seconds to Duration parsing is taken care by jackson, but as in other operators the time duration input is in the ISO-8601 format hence kept the same for consistency.